src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -276,4 +267,31 @@ protected void setMeasureReviewMetaDataForDraft(Measure measure) {
       measure.getReviewMetaData().setLastReviewDate(null);
     }
   }
+
+  private void saveMeasureBundle(Measure savedMeasure, String accessToken, String username) {
+    Export export =
+        Export.builder()
+            .measureId(savedMeasure.getId())
+            .measureBundleJson(
+                fhirServicesClient.getMeasureBundle(savedMeasure, accessToken, "export"))
```

Comment:
I feel like the thing we're saving should probably be generated outside of this function and passed in as a parameter. It's a bit misleading for a function called `saveMeasureBundle` to also make an external call to generate the thing to save.

Reply:
Yeah that make sense, I only created this method to fix the checkStyle complaining about number of lines per method.